src/comp/comp.py

```diff
@@ -24,48 +24,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [person.name for person in humans if person.name.startswith('D')]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [person.name for person in humans if person.name.endswith('e')]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [person.name for person in humans if any(person.name.startswith(item) for item in ['C', 'D', 'E', 'F', 'G'])]
```

Comment:
Could do it this way instead:
`c = [person.name for person in humans if any(person.name.startswith(['C', 'D', 'E', 'F', 'G'])]`